python/splitter.py

```diff
@@ -35,8 +50,10 @@ def itemize_participant(percent_added):
         if is_number(item):
             items.append(item)
             print(items)
+        elif item == "":
+            completed = True
         else:
-             completed = True
+            print("Please enter the item cost as a number using digits 0-9")
```

Comment:
The digits comment may confuse users what if you want to put 12.95 (".")

Reply:
Those are all still digits, it really just means to not type a number as words like twenty five. 12.95 has digits 1, 2, 9, and 5, all within the decimal range of 0-9. If you feel like it doesn't make sense though, then maybe it's enough to just ask them to enter a number and not mention digits at all

Reply:
yea i think the number is fine without digits